src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig

```diff
@@ -1460,7 +1460,7 @@
       {% if not child.rendered and child.vars.name != '_footer_buttons' and child.vars.name != '_token' %}
       <div role="tabpanel" class="form-contenttab tab-pane container-fluid{% if firstRenderedChild %} active{% endif %}" id="{{ child.vars.id }}-tab">
         {% if child.vars.label_tab is defined %}
-          {{ form_label(child, child.vars.label_tab) }}
+          {{ form_label(child, child.vars.label_tab, { required: false }) }}
```

Comment:
```suggestion
          {# We force required false here so that the required asterix is never shown on the tab title #}
          {{ form_label(child, child.vars.label_tab, { required: false }) }}
```